projects/core/src/process/store/selectors/process.selectors.ts

```diff
@@ -0,0 +1,48 @@
+import { MemoizedSelector, createSelector } from '@ngrx/store';
+
+import { StateWithProcess } from '../process-state';
+import { entityStateSelector } from '../../../state/utils/entity-loader/entity-loader.selectors';
+import { LoaderState } from '../../../state/utils/loader/loader-state';
+import {
+  loaderLoadingSelector,
+  loaderSuccessSelector,
+  loaderErrorSelector,
+} from '../../../state/utils/loader/loader.selectors';
+
+import { getProcessState } from './feature.selector';
+
+export function getProcessStateFactory<T>(
+  updateStateName: string
+): MemoizedSelector<StateWithProcess<T>, LoaderState<T>> {
+  return createSelector(
+    getProcessState(),
+    details => entityStateSelector(details, updateStateName)
```

Comment:
Can we discuss the var naming.  I't not clear for me what neither 'details' or 'updateStateName' are.

is 'updateStateName' = the process name?  If so, do you think it would improve readability to rename the var to `processName` ?

Reply:
That's a copy paste left over. Good catch